src/main/java/beam/analysis/summary/VehicleTravelTimeAnalysis.java

```diff
@@ -20,19 +30,47 @@ public void processStats(Event event) {
                     Double.parseDouble(eventAttributes.get(PathTraversalEvent.ATTRIBUTE_DEPARTURE_TIME)));
 
             secondsTraveledByVehicleType.merge(vehicleType, hoursTraveled, (d1, d2) -> d1 + d2);
+
+
+            if (vehicleType.contains("Car") || vehicleType.contains("BUS")){
```

Comment:
We have a number of different ways to handle something like this. @colinsheppard is there something I missed, or should I open an issue to try to create a static way to check vehicle type? Probably from BeamVehicleType.scala. This code is fine atm - I'm just using it as a reference point at this time.

Reply:
mode should have been compared here @rajnikantsh , not vehicleType